internal/database/migrations/migrations.go

```diff
@@ -14,23 +11,19 @@ func CreateTables(db *gorm.DB) error {
 }
 
 func Seed(db *gorm.DB) error {
-	rand.Seed(time.Now().Unix())
-	randNum := rand.Int()
+	babbler := babble.NewBabbler()
+	babbler.Count = 1
 
 	seed := models.Author{
-		FirstName:     "Grace " + strconv.Itoa(randNum),
-		LastName:      "Hopper",
-		Bio:           "pioneer",
+		FirstName:     babbler.Babble(),
```

Comment:
Can we just use https://github.com/dmgk/faker or similar? We can also easily reuse for testing then.

Reply:
you are right faker is a pretty good package. I will try integrating it